certified-connectors/AzureAD/apiDefinition.swagger.json

```diff
@@ -316,14 +316,13 @@
             "format": "int32"
           },
           {
-            "name": "$skip",
+            "name": "$skiptoken",
```

Comment:
How will user get this?

Reply:
Will not. See the other comment in the main PR thread. I also submitted a PR to revert the $skip token.